src/Actions/babylon.actionManager.ts

```diff
@@ -319,6 +319,22 @@
         }
 
         /**
+         * Unregisters an action to this action manager
+         * @param {BABYLON.Action} action - the action to be unregistered
+         * @return {Boolean}
+         */
+        public unregisterAction(action: Action): Boolean {
+            var index = this.actions.indexOf(action);
+            if (index !== -1) {
+                this.actions.splice(index, 1);
+                ActionManager.Triggers[action.trigger] -= 1;

```

Comment:
You may need to add this:
                if (ActionManager.Triggers[action.trigger] === 0) {
                    delete ActionManager.Triggers[action.trigger]
                }

so we can refactor this piece of code from the ActionManager.dispose